### PR TITLE
Fix color augmentation in TensorFlow object detection

### LINKED ISSUE #3148: Object detector doesn't find matches
Hi, I have setup the current version of turicreate in linux and tried training a model for object detection. The training completes but the model doesn't find objects in test images, either trying predict() or exporting the model to the sample iOS app.

I have checked the training images and annotations are correct using util.draw_bounding_boxes() and it looks good.

I also tried the same data in the MakeML app, that seems to use turicreate 5.2.1 the model from that works correctly with their test tool and also in iOS.

Does anyone have any clue how to debug this? When training the loss is decreasing by a similar amount as seen in the MakeML app. I would rather be able to run it on my own hardware. Thanks.

### PR DESCRIPTION
**Pull request by TobyRoseman**
Fixes #3148 

#3093 incorrectly set the range for saturation and contrast. This reverts to the previous behavior.

### CODE REVIEW

**Comment by TobyRoseman** [comment]
Internal tests pass.

Benchmarking show accuracy is inline with 6.1 release for two class datasets. For a dataset with six classes accuracy is more than three times as good.
